### ISSUE TITLE
You leaked your credentials

### ISSUE THREAD

**Issue by PhantomKnight287** [Hemant-0000/Image-Uploader#3]
# You're not supposed to push your credentials to a public repository 🗿 
https://github.com/hemantwasthere/Image-Uploader/blob/main/firebase.js

# Please go through this 
https://nextjs.org/docs/basic-features/environment-variables

**Comment by hemantwasthere** [comment]
Yeah i know, that was a huge mistake, can you correct this @PhantomKnight287 and thanks for opening the issue

**Comment by PhantomKnight287** [comment]
Just deleting the file won't do anything. You'll have to delete the commit history.

**Comment by hemantwasthere** [comment]
Can you send me a resource for deleting the commit history
